api/api.go

```diff
@@ -207,6 +207,7 @@ func New(requiredUserAgent string, requiredPassword string, cs modules.Consensus
 		router.GET("/renter", api.renterHandlerGET)
 		router.POST("/renter", RequirePassword(api.renterHandlerPOST, requiredPassword))
 		router.GET("/renter/contracts", api.renterContractsHandler)
+		router.GET("/renter/contract/*contractid", api.renterContractDetailHandler)
```

Comment:
@RNabel: I think these are the fields we should add to `api.RenterContract`:

- StartHeight
- HostKey
- Spending (upload, download, storage)
- Costs (TotalCost, Fees)

It is technically possible to extract `HostKey` from `LastTxn` already, but it's a bit of a chore.